utils/test/conversion_tests/ConversionTestSuite.py

```diff
@@ -0,0 +1,146 @@
+# coding: utf-8
+'''
+-----------------------------------------------------------------------------
+Copyright 2016 Esri
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-----------------------------------------------------------------------------
+
+==================================================
+ConversionTestSuite.py
+--------------------------------------------------
+requirments:
+* ArcGIS Desktop 10.X+ or ArcGIS Pro 1.X+
+* Python 2.7 or Python 3.4
+author: ArcGIS Solutions
+company: Esri
+==================================================
+description:
+This test suite collects all of the tests in the Conversion toolset within the Military Tools toolbox:
+
+
+==================================================
+history:
+5/10/2016 - JH - initial creation
+==================================================
+'''
+
+import logging
+import unittest
+import Configuration
+
+TestSuite = unittest.TestSuite()
+
+def getConversionTestSuites():
+    ''' Add all of the tests in ./conversion_tests to the test suite '''
+    
+    convertCoordinatesDesktopTests = ['test_convert_coordinates_desktop']
+    convertCoordinatesProTests = ['test_convert_coordinates_pro']
+    tableToTwoPointDesktopTests = ['test_table_to_twopointline_desktop']
+    tableToTwoPointProTests = ['test_table_to_twopointline_pro']
+    tableToEllipseDesktopTests = ['test_table_to_ellipse_desktop']
+    tableToEllipseProTests = ['test_table_to_ellipse_pro']
+    tableToLineOfBearingDesktopTests = ['test_table_to_lineofbearing_desktop']
+    tableToLineOfBearingProTests = ['test_table_to_lineofbearing_pro']
+    tableToPointDesktopTests = ['test_table_to_point_desktop']
+    tableToPointProTests = ['test_table_to_point_pro']
+    tableToPolygonDesktopTests = ['test_table_to_polygon_desktop']
+    tableToPolygonProTests = ['test_table_to_polygon_pro']
+    tableToPolylineDesktopTests = ['test_table_to_polyline_desktop']
+    tableToPolylineProTests = ['test_table_to_polyline_pro']
+    
+    if Configuration.DEBUG == True:
+        print("   ConversionTestSuite.getConversionTestSuites")
+        
+    Configuration.Logger.info("Adding Conversion tests including: ")
+    
+    if Configuration.Platform == "DESKTOP":
```

Comment:
Is there a reason for these 2 options "Desktop" vs. "Pro" (the set of tests looks the same)